docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/arc_automanage_machine_configuration_custom_linux/_index.md

```diff
@@ -0,0 +1,399 @@
+
+ $job | Wait-Job | Receive-Job
+```
+
+Check policy compliance by following these steps:
+
+- In the Azure Portal, navigate to *Policy* -> *Compliance*
+- Set the scope to the resource group your instance of ArcBox is deployed to
+- Filter for *(AzureArcJumpstart) [Linux] Custom configuration*
+
+![Screenshot of Azure Portal showing Azure Policy compliance](./02.png)
+
+It may take 15-20 minutes for the policy remediation to be completed.
+
+Get a Machine Configuration specific view by following these steps:
+
+- In the Azure Portal, navigate to *Azure Arc* -> *Servers*
+- Click on Arcbox-Ubuntu-01 -> Machine Configuration
+- If the status for *Arcbox-Ubuntu-01/AzureArcJumpstart_Linux* is not *Compliant*, wait a few more minutes and click *Refresh*
+
+![Screenshot of Azure Portal showing Azure Machine Configuration compliance](./03.png)
+
+Click on *Arcbox-Ubuntu-01/AzureArcJumpstart_Linux* to get a per-resource view of the compliance state in the assigned configuration
+
+![Screenshot of Azure Portal showing Azure Machine Configuration compliance](./04.png)
+
+### Verify that the operating system level settings are in place
+
+Login to Arcbox-Ubuntu-01 by running the below command
+- Enter the password **ArcDemo123!!** when prompted
+
+```powershell
+Enter-AzVM -ResourceGroupName $ResourceGroupName -Name Arcbox-Ubuntu-01 -LocalUser arcdemo
+```
+
+Verify that the packages **hello** and **nginx** are installed by running ```apt list --installed | grep 'hello\|nginx/focal'```
+
+![Screenshot of ArcBox-Ubuntu-01](./05.png)
+
+You can also verify that nginx is running and available by accessing http://10.10.1.103/ from Microsoft Edge on the ArcBox Client virtual machine.
+
+![Screenshot of nginx](./26.png)
+
+If you want to evaluate how remediation works, try to make one of the above configuration settings non-compliant by, for example, removing the package hello: ```sudo apt remove hello```
+
+Trigger a [manual evaluation](https://learn.microsoft.com/en-us/powershell/module/az.policyinsights/start-azpolicycompliancescan?view=azps-9.4.0) or wait until the next policy evaluation cycle has completed and observe that the policy is now non-compliant.
+
+Next, perform the steps outlined in [Create a remediation task](https://learn.microsoft.com/en-us/azure/governance/policy/how-to/remediate-resources?tabs=azure-portal#create-a-remediation-task) for the ArcBox-policies to bring the machine back into compliance.
+
+To learn more, check out [Remediation options for machine configuration](https://learn.microsoft.com/en-gb/azure/governance/machine-configuration/machine-configuration-policy-effects) in the documentation.
+
+## Summary
+
+In this scenario you have performed the following tasks:
+
+- [Created a package artifact](https://learn.microsoft.com/en-us/azure/governance/machine-configuration/machine-configuration-create) for machine configuration.
```

Comment:
Check all your URLs with the VSCode extension HTTP/s and relative link checker

![image](https://user-images.githubusercontent.com/96136892/231436374-dcf02264-fbee-46ee-aec5-8cc88160f93f.png)

Reply:
Resolved in https://github.com/microsoft/azure_arc/pull/1767/commits/3bcd006e6ccc5db8b1bf9029e20434735788bd9f